static/css/style.css

```diff
@@ -227,6 +229,15 @@ html .all-auth-container {
     color: var(--p-orange);
 }
 
+.enroll-hackathon-card, .enroll-hackathon-card form {
+    width: fit-content;
+    display: inline-block;
+}
+.enroll-hackathon-card form button#enroll-part {
+    margin: 0px!important;
```

Comment:
`margin: 0 !important;`
no `px` needed, and space required

Reply:
Fixed css as suggested.